mon-pix/app/controllers/campaigns/skill-review.js

```diff
@@ -14,16 +14,16 @@ export default class SkillReviewController extends Controller {
     'model.{campaignParticipation.campaignParticipationResult.badge,campaignParticipation.campaignParticipationResult.areBadgeCriteriaFulfilled}'
   )
   get shouldShowBadge() {
-    const badge = this.get('model.campaignParticipation.campaignParticipationResult.badge.content');
-    const areBadgeCriteriaFulfilled = this.get('model.campaignParticipation.campaignParticipationResult.areBadgeCriteriaFulfilled');
+    const badge = this.model.campaignParticipation.get('campaignParticipationResult.badge.content');
+    const areBadgeCriteriaFulfilled = this.model.campaignParticipation.get('campaignParticipationResult.areBadgeCriteriaFulfilled');
     return (!_.isEmpty(badge) && areBadgeCriteriaFulfilled);
   }
 
   @action
   shareCampaignParticipation() {
     this.set('displayErrorMessage', false);
     this.set('displayLoadingButton', true);
```

Comment:
Pourquoi ici, on ne fait pas 
```suggestion
    this.displayErrorMessage = false;
    this.displayLoadingButton = true;
```